kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -71,12 +71,12 @@ public BigQueryRecordConverter(boolean shouldConvertDoubleSpecial) {
    *                           consist of both a name and a value.
    * @return The result BigQuery row content.
    */
-  public Map<String, Object> convertRecord(SinkRecord kafkaConnectRecord) {
-    Schema kafkaConnectSchema = kafkaConnectRecord.valueSchema();
-    Object kafkaConnectValue = kafkaConnectRecord.value();
+  public Map<String, Object> convertRecord(SinkRecord kafkaConnectRecord, boolean convertKey) {
```

Comment:
nit: maybe cleaner to have 2 functions `convertKeyRecord()` & `convertValueRecord()`?

Reply:
Updated!